src/Dfc.CourseDirectory.Web/Controllers/VenuesController.cs

```diff
@@ -223,38 +234,7 @@ public async Task<IActionResult> AddVenueManualAddress(string Id)
 
             return View(viewModel);
         }
-        [Authorize]
-        public async Task<IActionResult> EditVenue(string Id)
```

Comment:
I think this should be replaced with a redirect to the new url in case any users have bookmarked etc.

Reply:
Done